ts/components/services/ServicesSectionsList.tsx

```diff
@@ -95,6 +99,23 @@ class ServicesSectionsList extends React.PureComponent<Props> {
     );
   }
 
+  // View for empty list in National and All tabs
+  private othersListEmptyComponent() {
```

Comment:
```suggestion
  private listEmptyComponent() {
```
since the special case is only for "local"